WordPress/Classes/Utility/In-App Feedback/InAppFeedbackPromptCoordinator.swift

```diff
@@ -40,11 +40,11 @@ final class InAppFeedbackPromptCoordinator: InAppFeedbackPromptPresenting {
             message: Strings.FeedbackAlert.message,
             preferredStyle: .alert
         )
-        let yes = UIAlertAction(title: Strings.FeedbackAlert.yes, style: .default) { [weak self] _ in
-            self?.handlePositiveFeedback(in: controller)
+        let yes = UIAlertAction(title: Strings.FeedbackAlert.yes, style: .default) { _ in
+            self.handlePositiveFeedback(in: controller)
         }
```

Comment:
This coordinator is released once the first alert shown ( see MySiteOverlaysCoordinator ), so `self` in this case is `nil`, because no other object is retaining this in-app ratings coordinator. Removing `[weak self]` doesn't cause a memory leak because the coordinator doesn't retain the `UIAlertViewController`.